src/dj_notebook/management/commands/shell_plus.py

```diff
@@ -0,0 +1,20 @@
+from django_extensions.management.commands import shell_plus
```

Comment:
I think I did not define the scope of this correctly. It is not meant to warn off on django-extension's `shell_plus`, rather to detect if dj-notebook is being run in production.